src/main/java/com/datapipe/jenkins/vault/VaultAccessor.java

```diff
@@ -17,25 +19,22 @@
     private transient Vault vault;
 
     private transient VaultConfig config;
+    private transient SslConfig sslConfig;
+    private transient VaultCredential credential;
 
-
-    public void init(String url) {
-        init(url, null, false);
-    }
-
-    public void init(String url, boolean skipSslVerification) {
-        init(url, null, skipSslVerification);
-    }
-
-    public void init(String url, VaultCredential credential) {
-        init(url, credential, false);
+    public VaultAccessor() {
+        config = new VaultConfig();
+        sslConfig = new SslConfig();
     }
 
-    public void init(String url, VaultCredential credential, boolean skipSslVerification) {
+    public void init(VaultConfiguration vaultConfig, VaultCredential credential) {
```

Comment:
I'd prefer if VaultConfiguration could pass a VaultConfig to VaultAccessor.

I would hate to breaking binary compatibility with https://github.com/jenkinsci/hashicorp-vault-pipeline-plugin again until we can get the two plugins merged.